assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -67,7 +67,7 @@ const PRESETS = [
         'Heading 2',
         'web-stories'
       )}</span>`,
-      fontWeight: FONT_WEIGHT.BOLD,
+      fontWeight: FONT_WEIGHT.MEDIUM,
```

Comment:
This property is actually completely ignored now (and has been for quite a while). The font-weight has to be supplied inline in the `content` HTML above - and that's still set to 700. Please remove this property from all the presets and fix the inline property in the HTML instead, thanks!

Reply:
Looks like this isn't ignored in the actual preview button content, but is ignored for rendering the element.